olm-catalog/serverless-operator/index/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM openshift/origin-base as builder
```

Comment:
Why not just start with `FROM quay.io/operator-framework/opm:latest`? 
This is what we end-up anyway.